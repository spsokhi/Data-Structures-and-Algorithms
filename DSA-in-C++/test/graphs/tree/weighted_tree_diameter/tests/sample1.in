8
0 1 5
1 2 3
2 3 1
1 4 2
4 7 4
1 5 7
2 6 5
