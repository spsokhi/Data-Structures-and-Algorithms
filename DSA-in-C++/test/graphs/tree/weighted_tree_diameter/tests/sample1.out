15 4
5 1 2 6
